Сверстана секция perfomance

s-gusterev opened the pull request.

s-gusterev committed:
сверстаны основные блоки

s-gusterev committed:
сверстаны основные блоки

s-gusterev committed:
исправил margin-bottom у абзацев

s-gusterev committed:
добавил hover для карточки

s-gusterev committed:
добавил hover для автора в карточке

s-gusterev committed:
добавил hover для кнопки билеты

s-gusterev committed:
сверстал основной блок с контентом

nadyaiva reviewed (approved):
Здорово сделано - грид, семантические теги, анимация. Я проблем не увидела)

pankiale commented:
resolve conflicts before merging

Commit:
Merge branch 'develop' into feature/perfomance

Commit:
небольшие исправления

1. Добавлен марджин от хедера
2. Убран паддинг в конце (у секции с галереей уже есть марджин)
3. Добавлены альтернативные варианты отрисовки шрифтов
4. Исправлено typo в слове performance
